migrations/versions/7bb5891a9f2e_lowercase_email.py

```diff
@@ -18,7 +18,7 @@
 
 
 def upgrade():
-    op.execute("UPDATE users SET deleted_at = current_timestamp, _email = concat(_email, '_') FROM (SELECT lower(_email) FROM users GROUP BY lower(_email) HAVING COUNT(*) > 1) as email where lower(_email) != _email;",
+    op.execute("UPDATE users SET deleted_at = current_timestamp, _email = concat(_email, '_') where _email not in (SELECT DISTINCT ON (upper(_email)) _email FROM users);",
```

Comment:
Query is correct but it deletes the users with lowercase names rather than uppercase

Reply:
@iamareebjamal I tried to do it using Order by as you suggested, but in the distinct on query, I can only order by using upper(_email), which does not solve this issue.

Reply:
Can you order by _email?

Reply:
@iamareebjamal It shows error to order it by using upper(_email) only. Distinct on and order by are somewhat related.

Reply:
OK, will see what can be done. Merging for now, query can be improved till when it will run on production DB

Reply:
@iamareebjamal Ok.